### ISSUE TITLE
Library vs Package

### ISSUE THREAD

**Issue by hangtime79** [IBMPredictiveAnalytics/Watson-Personality-Insights#1]
Should this not be a request to check for the HTTR and JSONLITE packages not libraries?

**Comment by gfilla** [comment]
Hi - although I did not develop this extension I just reviewed the source code.  I think the library you are referencing is the name given to a variable and function, but it is installing the package not a library.  The function library() for httr and jsonlite loads a package in R.  Are you experiencing difficulty with the extension and packages not loading?

**Comment by hangtime79** [comment]
Yes, when I tried to do the original install, I was repeatedly getting errors. It was not until I changed the code to reference the package(httr) and put in to place the code to test for the package that I was able to get this to work.

**Comment by avikbose** [comment]
First time trying to execute Watson personality insight node from SPSS modeller 16 using IBM bluemix credential - there is a demo code in github using jsonlite & httr. execution throws server exception java.lang.IllegalArgumentException. How do I debug?

**Comment by gfilla** [comment]
@avikbose Sorry about delay in reply.

Can you please send the console output in the node?  That will help us debug.

Best - Greg

![image](https://cloud.githubusercontent.com/assets/6307720/10802350/6cc75672-7d8a-11e5-99e4-020d0bae1f36.png)

**Comment by jasonmburke** [comment]
I too had an issue and resolved it by manually installing the R6 package which was a dependency for HTTR.
